styles/globals.css

```diff
@@ -1,9 +1,33 @@
+@font-face {
+  font-family: "DIN";
+  src: url("../public/fonts/DIN/DIN-Regular.woff") format("woff");
+}
+@font-face {
+  font-family: "SignPainter-HouseScript";
+  src: url("../public/fonts/SignPainter/SignPainter.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Copperplate";
+  src: url("../public/fonts/Copperplate/Copperplate.woff") format("woff");
+}
+
+:root {
+  --accent-color: #ff4064;
+  --second-accent-color: #ffb700;
```

Comment:
Small suggestion: Maybe name these `primary` and `secondary`, so it matches with the other variables

Reply:
Yes tx ;)